app/src/main/java/de/westnordost/streetcomplete/quests/seasonal/AddSeasonal.kt

```diff
@@ -0,0 +1,32 @@
+package de.westnordost.streetcomplete.quests.seasonal
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.osm.updateWithCheckDate
+import de.westnordost.streetcomplete.quests.YesNoQuestForm
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddSeasonal : OsmFilterQuestType<Boolean>() {
```

Comment:
As this is about grit bins only, why not name it that way?

Reply:
Probably because it could be extended... could be useful e.g. for mobile toilets that are stationary, but removed in winter (I know some).

Reply:
That's fair. However, for mobile toilets for example, the question is somewhat unanswerable. Because you would have to check... I don't know, once every month to see if it is still there, and when you made 12 ✔️ , then you can answer "Yes, it's always there". For grit bins (winter) and outdoor seating (summer), you at least know which time of the year you have to look out for.

Reply:
Changed in [`d57dcdd`](https://github.com/streetcomplete/StreetComplete/pull/4754/commits/d57dcdd7311e85f7916cc43ba00f849f3a21ef07).